### PR TITLE
V2: Update readme

### PR DESCRIPTION
**Pull request by Zmoki**

### CODE REVIEW

**Comment by Zmoki** [comment]
[Widget: выпустить патч второй ветки](https://trello.com/c/tuO8b6vi/38-widget-%D0%B2%D1%8B%D0%BF%D1%83%D1%81%D1%82%D0%B8%D1%82%D1%8C-%D0%BF%D0%B0%D1%82%D1%87-%D0%B2%D1%82%D0%BE%D1%80%D0%BE%D0%B9-%D0%B2%D0%B5%D1%82%D0%BA%D0%B8)

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+### Enviroment
+
+You need a working Ruby 2.0.0 and above environment with [Bundler](http://bundler.io).
+
+#### Vagrant
+
+If you don't want to raise the environment on your machine, you can use [Vagrant](https://www.vagrantup.com/).
+Just [install Vagrant](https://www.vagrantup.com/docs/installation/) and [VirtualBox](https://www.virtualbox.org/) or [other provider](https://www.vagrantup.com/docs/getting-started/providers.html).
+
+After open command line, go to this folder and run:
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Then, navigate to *that???* directory via your command line, and run:

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    vagrant up
+    vagrant ssh
+
+#### Local environment (without Vagrant)
+
+Inside folder run
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Navigate to your local environment directory and run:

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  The version is specified in `lib/uploadcare-widget/version.rb`.
+
+### Development
+
+Go to `test/dummy/`. There is a simple Rails app. Run it:
+
+    bundle exec rails server
+
+Open http://127.0.0.1:3000/. Follow any link.
+There's going to be a widget or three. Edit code and reload the page :-)
```
Comment (dayton1987) [inline DEVELOPMENT.md]: wut? the phrase goes like "a widget or two."
"Edit code and..." >> Modify code? Mutate code?

File: DEVELOPMENT.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+
+### Testing
+
+[Jasminerice](https://github.com/bradphelan/jasminerice)
+installed under the `test/dummy/` Rails app.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: Use Jasminerice for testing purposes: install it under the <name> test app.

File: DEVELOPMENT.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+To run tests in your browser go to http://127.0.0.1:3000/jasmine.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: browser, go

File: DEVELOPMENT.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+For more information, see
+[jasminerice docs](https://github.com/bradphelan/jasminerice).
+
+#### guard-jasmine
+
+To run tests in a terminal you must first
```
Comment (dayton1987) [inline DEVELOPMENT.md]: terminal, you must

File: DEVELOPMENT.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+[install phantomjs](https://github.com/guard/guard-jasmine#phantomjs).
+
+Then, there are two options:
+
+  - run `bundle exec guard start`
+  - run `bundle exec guard-jasmine`
+
+The first one is for continuous tests execution,
+the second one runs tests just once.
+Both should be executed from the `test/dummy/` directory.
```
Comment (dayton1987) [inline DEVELOPMENT.md]: The difference is that the first option goes with the continuous test execution while the second one will run the tests once. Both commands should be run under the <name> directory.

File: README.markdown [hunk 8]
```diff
@@ -1,144 +1,180 @@
...
+---
 
-```html
-<script>
-  // Widget settings
-  UPLOADCARE_PUBLIC_KEY = 'your_public_key';
-</script>
-<script src="https://ucarecdn.com/widget/x.y.z/uploadcare/uploadcare.full.min.js" charset="utf-8"></script>
-```
+**Note: we have a new major version of the widget. [Here’s how][widget-docs-migration-v2-v3] you migrate from v2 to v3.**
```
Comment (dayton1987) [inline README.markdown]: Note: there currently is a new major version of the widget. Here's how you migrate from v2 to v3.

File: README.markdown [hunk 9]
```diff
@@ -1,144 +1,180 @@
...
-To run tests in a terminal you must first 
-[install phantomjs](https://github.com/guard/guard-jasmine#phantomjs).
+```html
+<script>
+  UPLOADCARE_PUBLIC_KEY = 'YOUR_PUBLIC_KEY';
+</script>
+```
 
-Then you have two options:
+Your secret key is not required for the widget; (it’s quite careless for your
```
Comment (dayton1987) [inline README.markdown]: (it's >> it's

**Pull request merged**